lib/Autocomplete.js

```diff
@@ -103,18 +103,9 @@ let Autocomplete = React.createClass({
     if (this.keyDownHandlers[event.key])
       this.keyDownHandlers[event.key].call(this, event)
     else {
-      const { selectionStart, value } = event.target
-      if (value === this.props.value)
```

Comment:
This was previously `value === this.state.value`, which caused the next block of code to execute every time, and breaking the workaround which made user-defined selections not break. When I "fixed" it by changing it to it's current form I overlooked the fact that `event.target.value` is always equal to `this.props.value`, since React prevents the input's value from ever changing.